src/mme-app/procedureStats.cpp

```diff
@@ -74,3 +74,5 @@ int ProcedureStats::num_of_erab_mod_ind_received = 0;
 int ProcedureStats::num_of_erab_mod_conf_sent = 0;
 int ProcedureStats::num_of_erab_mod_ind_timeout = 0;
 int ProcedureStats::num_of_abort_erab_mod_indication = 0;
+int ProcedureStats::num_of_req_sent_to_dns = 0;
```

Comment:
I dont see a ccode where num_of_req_sent_to_dns is incremented. Can you please pass me reference ?

Reply:
Sorry, I have added it into req_to_dns and move   ProcedureStats::num_of_processed_dns_response ++; from req_to_dns  to
process_dns_resp function